### DISCUSSION TITLE
failed to build for linux_arm64 / usage: link

### DISCUSSION THREAD

**Discussion by greenpau** [goreleaser/goreleaser#3411] [Q&A] [answered]
The goreleaser fails with the following error.

```
   ⨯ release failed after 1m21s               error=failed to build for linux_arm64: exit status 2: # github.com/greenpau/ovn_exporter/cmd/ovn_exporter
usage: link [options] main.o
  -B note
    	add an ELF NT_GNU_BUILD_ID note when using ELF
```

Job: https://github.com/greenpau/ovn_exporter/actions/runs/3137457689/jobs/5095646150

`.goreleaser`: https://github.com/greenpau/ovn_exporter/blob/main/.goreleaser.yml

Github action: https://github.com/greenpau/ovn_exporter/blob/main/.github/workflows/build.yml

![image](https://user-images.githubusercontent.com/3826416/192592214-99cd49da-be72-48a4-9c54-c2b46124bf16.png)

**Comment by caarlos0** [comment, marked as answer]
its the `\` in your ldflags, if you build that go build command manually it won't work as well.

**Reply by greenpau** [reply]
Thank you, @caarlos0 ! Fixed and worked.
